### PR TITLE
Add community documentation

### PR DESCRIPTION
**Pull request by RichardLitt**
Dearest humans,

You are missing some important community files. I am adding them here for you!

Here are some things you should do manually before merging this Pull Request:

- [ ] Update the Contributing guide to include any repository-specific requests, or to point to a global Contributing document.
- [ ] Check that the license name and year is correct. I've added the MIT license, which should suit most purposes.
- [ ] Check the email in the Code of Conduct. We've added in richard.littauer@gmail.com.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,5 +23,9 @@
     "semantic-release": "^8.0.0",
     "standard": "^10.0.3",
     "tap": "^10.7.2"
-  }
-}
+  },
+  "keywords": [],
+  "contributors": [
```
Comment (gr2m) [inline package.json]: so cool! I didn’t know that contributors is a [field in package.json](https://docs.npmjs.com/files/package.json#people-fields-author-contributors)!

Comment (RichardLitt) [reply package.json]: yeah! Also - I'll be able to fill this out with https://github.com/RichardLitt/name-your-contributors :D

Comment (gr2m) [reply package.json]: very, very cool! That could be a GItHub App all by itself! Keeping contributors up-to-date
You might also be interested in joining forces with https://github.com/kentcdodds/all-contributors/issues/58
I’d love to see that, and happy to help

**Pull request merged**